src/Parsers/queryNormalization.h

```diff
@@ -10,6 +10,7 @@
 
 namespace DB
 {
+template <bool with_names>
```

Comment:
I'd suggest you rename template parameter here (and in the method below) as well

Reply:
Done